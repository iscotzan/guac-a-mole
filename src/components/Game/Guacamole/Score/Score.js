import * as React from "react";

const Score = props => {
  return (
    <div className="scoreContainer">
      <h1 data-shadow={props.score}>  
        {props.score}
      </h1>
    </div>
  );
};

export default Score;
