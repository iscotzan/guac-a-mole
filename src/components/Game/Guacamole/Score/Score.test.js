import React from "react";
import Score from "./Score";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

const wrapper = shallow(<Score score={12345} />);
describe("<Score />", () => {
  it("should be defined", () => {
    expect(wrapper).toBeDefined();
  });

  it("renders without crashing", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
