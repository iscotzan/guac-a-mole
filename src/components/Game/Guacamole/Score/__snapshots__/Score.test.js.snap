// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Score /> renders without crashing 1`] = `
<div
  className="scoreContainer"
>
  <h1
    data-shadow={12345}
  >
    12345
  </h1>
</div>
`;
